changelog/upgrade-notes.md

```diff
@@ -6,6 +6,17 @@
 
 The deprecated `ol.animation` functions and `map.beforeRender()` method have been removed.  Use `view.animate()` instead.
 
+The deprecated `unByKey` function of the `ol.Observable` class has been removed.  Use the `ol.Observable.unByKey` static function instead:
```

Comment:
I think there is still room for confusion here.  The difference between a "function of a class" and a "static function" is too subtle.  I'd suggest this:

> The `unByKey()` method has been removed from `ol.Observable` instances.  Use the `ol.Observable.unByKey()` static function instead.